sys-power/acpilight/files/acpilight.initd

```diff
@@ -0,0 +1,45 @@
+#!/sbin/openrc-run
+# $Id$
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+state_dir=/var/lib/acpilight
+
+extra_commands="save restore"
+
+depend() {
+	need localmount
+	after bootmisc modules isapnp coldplug hotplug
+}
+
+restore() {
+	ebegin "Restoring brightness level"
+	if [ ! -r "${state_dir}/state" ] ; then
+		ewarn "No brightness level in ${state_dir}/state"
+		eend 0
+		return 0
+	fi
+	xbacklight "$(cat "${state_dir}/state")"
+	eend 0
```

Comment:
I'd say you should pass `$?` here. Otherwise you ignore errors and pretend that it succeeded when it fails.